packages/tools/devtools/devtools-browser-extension/src/background/BackgroundScript.ts

```diff
@@ -183,3 +183,21 @@ browser.runtime.onConnect.addListener((devtoolsPort: Port): void => {
 
 	devtoolsPort.onMessage.addListener(devtoolsMessageListener);
 });
+
+// Listener for TEST_GET_TAB_ID message.
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
```

Comment:
Similar to the previous comment, I don't think we need extra infrastructue for message passing?